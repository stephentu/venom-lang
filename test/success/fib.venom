def fibn(x::int) -> int = 
  if x == 0 or x == 1 then x;
  else fibn(x - 1) + fibn(x - 2); end
end
print (fibn(0));
print (fibn(1));
print (fibn(2));
print (fibn(5));
print (fibn(8));
print (fibn(10));
#print (fibn(30));
