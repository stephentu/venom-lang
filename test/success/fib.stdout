0
1
1
5
21
55
